HumanControls/Utils.cpp

```diff
@@ -1,19 +1,32 @@
 #include "Utils.h"
 
-static void Utils::setMotors(TShirtCannonPayload payload, float turn, float speed)
+#define MAX_DRIVE_SPEED 63
+
+void Utils::setMotors(TShirtCannonPayload &payload, float turn, float speed)
 {
+
     double left, right;
     uint8_t leftSpeed, rightSpeed, leftDir, rightDir;
+    int leftTrim, rightTrim;
 
-    left = -((-1 * speed) + (-1 * turn));
-    right = -(speed + (-1 * turn));
+    leftTrim = 25;
+    rightTrim = -25;
+
+    right = (-1 * turn) + (-1 * speed);
+    left = turn + (-1 * speed);
 
     leftDir = left < 0 ? 1 : 0;
     rightDir = right < 0 ? 1 : 0;
+    
+    turn = turn * turn * turn;
```

Comment:
We can use the `pow(a,b)` function, and use should use a constant to determine how much we want to curve this. This can be done later.